import_ldraw.py

```diff
@@ -298,18 +306,18 @@ def parse(self, filename):
                     if tmpdate != "":
                         tmpdate = tmpdate.split()
 
-                        # TODO What is this condition for?
-                        # le717 unable to find a case where it is hit.
-                        if (tmpdate[0] == "0" and
-                            len(tmpdate) >= 3 and
-                            tmpdate[1].lower() == "!ldraw_org" and
-                            "part" in tmpdate[2].lower() and
-                            self.part_count > 1
-                        ):
-                            self.subparts.append(
-                                [filename, self.mat, self.colour]
-                            )
-                            break
+                        # LDraw brick comments
```

Comment:
Sometimes the solution is really simple. In the .dat-files two types of header lines are considered:

     0 !LDRAW_ORG Part UPDATE 2003-02
Or:
     0 !LDRAW_ORG Subpart UPDATE 2003-02

We are only interested in a top-level part and not in sub-parts. This explains why making the check case-insensitive effectively changes the behavior.

The fact that when subpart ARE considered, certain parts break up in more elements,  immediately explains what this condition is for and makes it obvious that the naming of self.subparts is not ideal. It effectively should be called self.parts. 

One uses the term of subparts for subdivisions of parts where these subdivisions have NO relevance to a physical LEGO-builder [my interpretation of the discussions on LDraw.org]. 

So parts => should be visible as such in Blender; subparts => must be merged together to form a part and no reference of themselves should be visible in Blender.

Now that I understand this a bit better, I still have to figure out how to deal with an Unofficial_model. Oh joy! ;-)